docs/databases.rst

```diff
@@ -175,3 +175,60 @@ under `READ`:
 * There can only be one master database (not a standby database)
 * All standby databases must point to the same master database
 * If a master database is in this list, all standbys must point to this master
+
+Using standbys with the plproxy cluster
+.......................................
+The plproxy cluster needs some special attention since the queries are routed by plproxy and not by
+Django. In order to do this routing there are a number of additional pieces that are needed:
+
+1. Separate plproxy cluster configuration which points the shards to the appropriate standby node instead
+of the primary node.
+2. Duplicate SQL functions that make use of this new plproxy cluster.
+
+In order to maintain the SQL function naming the new plproxy cluster must be in a separate database.
+
+Steps to setup:
+
+1. Add all the standby shard databases to the Django `DATABASES` setting as described above.
+
+2. Create a new database for the standby plproxy cluster configuration and functions and add it to `DATABASES`
+as shown below:
+
+.. code-block:: python
+
+    DATABASES = {
+        'proxy_standby': {
+            ...
+            'PLPROXY': {
+                'PROXY_FOR_STANDBYS': True
+            }
+        }
+    }
+
+3. Run the Django migrations to create the SQL functions in the new standby proxy database.
+
+Routing queries to standbys
+---------------------------
+The configuration above makes it possible to use the standby databases however in order to actually
+route queries to them the DB router must be told to do so. This can be done it one of two ways:
+
+1. Via an environment variable
+
+.. code-block::
+
+    export READ_FROM_PLPROXY_STANDBYS=1
```

Comment:
what is the rationale behind doing an all or nothing approach versus a load balanced approach like we do for apps? (not that I think this is a terrible choice, just curious if its worth having multiple approaches for standbys)

Reply:
I did think about doing that and we could change it in future if we want but I thought that in this case we always want to read from standbys if they are available. The 'load balancing' we do for the main DB ended up being quite similar in since we a 12:1 ratio of reads to standbys vs master. I think we could have taken a similar approach there.

I made the environment variable option because for processes like the pillows I thought it would be easier to do that than change all the places that the pillows do reads (which granted isn't many). It also meant that it would be easy to turn off if needed or that the `DocumentStore` classes could still be used to read from primaries in other scenarios. 

I'm very open to other ideas here - this was one of the bits I was less sure about